library/src/main/java/README.md

```diff
@@ -12,5 +12,6 @@ Kotlin code emitted by the `bindings_ffi` crate in [libxmtp](https://github.com/
 4. Navigate to the [latest release](https://github.com/xmtp/libxmtp/releases) once the action completes
 5. Download the `LibXMTPKotlinFFI.zip` assets
 6. Unzip and then copy the jniLibs to `xmtp-android/library/src/main/jniLibs`
+7. NOTE: sometimes the library name gets updated and will cause failures the library name must be `uniffi_xmtpv3` not `xmtpv3` -- search for `findLibraryName` and make sure the correct name is returned.
```

Comment:
I don't know why when we bumped to uniffi version 28 that this started renaming the library. I did a deep dive today through the docs and don't see any reason why this would be happening https://mozilla.github.io/uniffi-rs/latest/kotlin/configuration.html. This is a work around until we can invest more time in it to figure it out.

Reply:
Re- lint failure, not sure if we should maybe re-add this line as well:

https://github.com/xmtp/xmtp-android/blame/9e47f24fd3b714be09e6fedfc7364c97a68d0c8a/library/src/main/java/README.md#L12

Reply:
Good call that one came back too. 😭